staging/src/k8s.io/sample-controller/README.md

```diff
@@ -0,0 +1,24 @@
+# sample-controller
+
+This repository implements a simple controller for watching Foo resources as
+defined with a CustomResourceDefinition (CRD).
+
+It makes use of the generators in [k8s.io/code-generator](https://github.com/kubernetes/code-generator)
+to generate a typed client, informers, listers and deep-copy functions. You can
+do this yourself using the `./hack/update-codegen.sh` script.
```

Comment:
Can you add in the README what files do I need to have before running the script and what files will be generated by the script?